### PR TITLE
Created for fans admin back. Made new endpoints, Mapper and MediatR methods. 

### PR DESCRIPTION
**Pull request by DanyilTerentiev**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions
#197

### CODE REVIEW

**Review by Dobriyr — APPROVED** [review]

**Review by NadiaKishchuk — APPROVED** [review]
Well done. Please, see my comments)
And also don`t you need the delete for this entity? Because I don`t see the handler for this here.

**Review by MaksBrat — APPROVED** [review]

**Review by Tysyatsky — APPROVED** [review]
New handlers are always great 👯‍♂️

File: Streetcode/Streetcode.BLL/MediatR/Sources/SourceLinkCategory/GetAll/IRequest.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Streetcode.BLL.MediatR.Sources.SourceLinkCategory.GetAll
+{
+    public interface IRequest<T1, T2>
+    {
+    }
+}
```
[the remark this answers is not in the document]
Comment (DanyilTerentiev) [reply Streetcode/Streetcode.BLL/MediatR/Sources/SourceLinkCategory/GetAll/IRequest.cs:6]: To be honest, I didn't write this piece of code, it seems to me that it has been generated automatically by mediatR when I wrote handler for getAll.

[the remark this answers is not in the document]
Comment (NadiaKishchuk) [reply Streetcode/Streetcode.BLL/MediatR/Sources/SourceLinkCategory/GetAll/IRequest.cs:6]: So, please delete it) and freely merge

**Pull request merged**